draft-ietf-quic-transport.md

```diff
@@ -1825,6 +1826,31 @@ Connection ID:
 
 : A 64-bit connection ID.
 
+## STOP_SENDING Frame {#frame-stop-sending}
+
+An endpoint may use a STOP_SENDING frame (type=0x0c) to communicate that
+incoming data is being discarded on receipt at application request.  This
+signals a peer to abruptly terminate transmission on a stream.  The frame is as
+follows:
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                        Stream ID (32)                         |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                        Error Code (32)                        |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+
+The fields are:
+
+Stream ID:
+: The 32-bit Stream ID of the stream being ignored.
+
+Error Code:
+: The reason the sender is ignoring the stream, or QUIC_NO_ERROR if not due to
+  an error.
```

Comment:
Isn't this always an application error code and never a QUIC one? If so, can we simply say this is for the application to specify and consume, as one of the app-specified error codes? I don't think you need a QUIC_NO_ERROR in this case.